src/rules/jsxUseTranslationFunctionRule.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright 2016 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Lint from "tslint";
+import * as ts from "typescript";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    public static TRANSLATABLE_ATTRIBUTES = new Set(["placeholder", "title", "alt"]);
```

Comment:
only one i can think of is href for translated routes.

Maybe make this `Set` be configurable in the tslint config for the `jsx-use-translation-function` options.